### ISSUE TITLE
Problems launching new projects from my user account

### ISSUE THREAD

**Issue by sharoncy** [Neural-Systems-at-UIO/RB-workbench#118]
I am not able to perform testing as new project fail to launch from my user account. 

I looked at the bucket via the dataproxy, it said I didn't have access rights. Is it something to do with permissions?

**Comment by ArdaBalkir** [comment]
I believe it is. Just started picking apart the project but certain buckets we have administrator role for and that's how we operate in them. And the newer ones created do have it. Perhaps @PolarBean knows more.

**Comment by PolarBean** [comment]
its really strange that it seems just to be happening on sharons account. Can you create a new project and bucket via the collab?

**Comment by sharoncy** [comment]
Yes I created a new project and bucket via the collab and added both of you as admins. I also uploaded some images with no problem. https://wiki.ebrains.eu/bin/view/Collabs/test-sharoncy/

**Comment by PolarBean** [comment]
really strange that its just a problem via the workbench. @ArdaBalkir you can successfully launch new projects and upload files presumably?

**Comment by ArdaBalkir** [comment]
Yep, I can.
